google/cloud/aiplatform/explain/lit.py

```diff
@@ -260,10 +361,17 @@ def create_lit_model(
             Optional. A string to choose what attribution configuration to
             set up the explainer with. Valid options are 'sampled_shapley'
             or 'integrated_gradients'.
+          model_id:
+            Optional. A string of the specific model in the endpoint to create the
+            LIT model from. If this is not set, any usable model in the endpoint is
+            used to create the LIT model.
         Returns:
             A LIT Model object that has the same functionality as the model provided.
     """
-    return _VertexLitModel(model, input_types, output_types, attribution_method)
+    if os.path.exists(model):
```

Comment:
Endpoint resource name would be a valid path. Unlikely it would exist, but this provides greater motivation to accept `aiplatform.Endpoint` so we can rely on type checking. Relating to the comment above, separating into a new function would remove the requirement to check.

Reply:
Separating to another function.